### PR TITLE
Fixes #88 - Add pull target which will work after destroy

### LINKED ISSUE #88: [Feature] Create a "pull" target to let someone update their containers from upstream
Even when using :latest in a container tag, compose does not automatically pull new versions. You must use `docker-compose pull`. I propose adding a mirror `make pull` that will update the containers, this will not be part of the `make quick-start` to ensure we don't overwrite containers someone is working on locally.

Docker-compose issue stating how you must use pull:  https://github.com/docker/compose/issues/3574

### PR DESCRIPTION
**Pull request by mcdan**

### CODE REVIEW

**Comment by jthomas** [comment]
@mcdan gentle nudge on this :) it'll be great to get this merged.

**Comment by mcdan** [comment]
@jthomas I don't have write access so I can't even add reviewers. @ddragosd can have a quick look at this?

**Comment by jthomas** [comment]
@mcdan I thought I'd left a comment before but it has disappeared now! I can review & merge if that update is done about the docs.

File: docker-compose/README.md [hunk 1]
```diff
@@ -146,6 +146,9 @@ Once the installation process has completed, you can check the `whisk.system` na
 ```
 wsk package list /whisk.system
 ```
+## Updating Oontainers
```
Comment (jthomas) [inline docker-compose/README.md]: `Updating Oontainers` :)

Comment (mcdan) [reply docker-compose/README.md]: 933a935 fixes that, already pushed.

**Comment by jthomas** [comment]
💯

**Pull request merged**